user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/Custom_Operator/Creating_Minus_Operator.md

```diff
@@ -38,6 +38,8 @@ The result will look like this:
   > [!TIP]
   > See [Activating soft-launch options](xref:Activating_Soft_Launch_Options).
 
+- Basic knowledge of GQI extension development
```

Comment:
Should there be a punctuation mark at the end? Since there are for the other prerequisites too.

Reply:
Indeed! Good point 🙂